nesta/core/orms/orm_utils.py

```diff
@@ -268,7 +268,29 @@ def filter_out_duplicates(db_env, section, database,
     try_until_allowed(Base.metadata.create_all, engine)
     Session = try_until_allowed(sessionmaker, engine)
     session = try_until_allowed(Session)
+    return _filter_out_duplicates(session, Base, _class, data, low_memory)
```

Comment:
refactored to accept `session` to fit directly into Russ's code. In the daps2 rewrite I'll completely change the arxlive pipeline so that it is more consistent with the rest of the codebase.